Objective-C/CBLReplicator.h

```diff
@@ -160,7 +160,7 @@ typedef struct {
  @param error error On return, the error if any.
  @return A  set of document Ids, each of which has one or more pending revisions
  */
-- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error;
+- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error NS_SWIFT_NOTHROW;
```

Comment:
better solution, and found recommended Apple way of doing 
https://developer.apple.com/documentation/swift/cocoa_design_patterns/about_imported_cocoa_error_parameters